src/nbl/video/CVulkanConnection.cpp

```diff
@@ -352,7 +357,9 @@ bool CVulkanConnection::endCapture()
         return false;
     }
 
-    m_rdoc_api->EndFrameCapture(RENDERDOC_DEVICEPOINTER_FROM_VKINSTANCE(m_vkInstance), NULL);
+    if (debugType == EDT_RENDERDOC)
+        m_rdoc_api->EndFrameCapture(RENDERDOC_DEVICEPOINTER_FROM_VKINSTANCE(m_vkInstance), NULL);
+    // no equivalent end frame capture for ngfx, ends captures on next frame delimiter
```

Comment:
leave this link in the comments https://www.reddit.com/r/GraphicsProgramming/comments/w0hl9o/graphics_debugger_record_before_first_frame/